Demo-Code.py

```diff
@@ -9,11 +9,26 @@ def main():
     Image.fromarray(rand_pic_np).save("result.png")
     print(ball_set)
 
-def pic_generator(yields_num, max_balls, min_diam, max_diam, bg_width, bg_height):
+def pic_generator(yields_num: int, max_balls: int, min_diam: int, max_diam: int, bg_width: int, bg_height: int):
+    """
+    :param yields_num: number of yields
+    :yield: get_random_pic(max_balls, min_diam, max_diam, bg_width, bg_height)  
```

Comment:
instead of writing the fuction you call give a brief description of the fuction return value